src/index.css

```diff
@@ -90,6 +90,17 @@ main {
   gap: 15rem;
 }
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225288148-aeb8901b-b4da-4e2e-9fd0-dc9698d622d0.png)

- [ ] Kindly update your sections as shown below to make your readme file useful 👍 for the new developers or viewers of your application 😊

## Install
```
npm install
```

## Usage
```
npm run build
npm start
```

## Run tests
```
npm test
```